.github/workflows/release.yml

```diff
@@ -13,6 +13,11 @@ jobs:
     outputs:
       upload_url: ${{ steps.create_release.outputs.upload_url }}
     steps:
+      - name: Get latest alan version
+        id: get_latest_alan
+        run: |
+          echo "::set-output name=alan_version::$(curl --silent "https://api.github.com/repos/alantech/alan/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')"
```

Comment:
are we grabbing the entire string with the `v` included?

Reply:
Yes, even tho in this case it is only informational.

![image](https://user-images.githubusercontent.com/5898639/110842206-16e22200-82a7-11eb-8394-4e38926f9a9f.png)

Reply:
perfect!